lib/puma/configuration.rb

```diff
@@ -133,6 +133,7 @@ class Configuration
       debug: false,
       early_hints: nil,
       environment: 'development'.freeze,
+      supported_http_methods: Puma::Const::DEFAULT_SUPPORTED_HTTP_METHODS,
```

Comment:
We're inside the `Puma` module so we can do this

```suggestion
      supported_http_methods: Const::DEFAULT_SUPPORTED_HTTP_METHODS,
```